addons/mail/models/mail_mail.py

```diff
@@ -587,7 +587,13 @@ def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
                             raise
                 if res:  # mail has been sent at least once, no major exception occurred
                     mail.write({'state': 'sent', 'message_id': res, 'failure_reason': False})
-                    _logger.info('Mail with ID %r and Message-Id %r successfully sent', mail.id, mail.message_id)
+                    _logger.info(
+                        "Mail with ID %r and Message-Id %r from %r to (redacted) %r successfully sent",
+                        mail.id,
+                        mail.message_id,
+                        tools.mail.email_anonymize(tools.email_normalize(msg['from'])),
```

Comment:
I don't think we need to anonymize the FROM (and for practical uses should not). This should always be a Odoo default (notification@ etc) or a email address from an internal user where information leaked should be less of a concern